arcade/standard/blockglitch_4/map.xml

```diff
@@ -441,33 +443,33 @@
-    <portal forward="all(not(portal-debounce), do_tp=1, tp_lane=6)" x="348" observers="never" />
-    <portal forward="all(not(portal-debounce), do_tp=1, tp_lane=7)" x="406" observers="never" />
-    <portal forward="all(not(portal-debounce), do_tp=1, tp_lane=8)" x="464" observers="never" />
-    <portal forward="all(not(portal-debounce), do_tp=1, tp_lane=9)" x="522" observers="never" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-9, tp_checkpoint=..8)" yaw="@0" pitch="@0" observers="never" destination="cp-9" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-8, tp_checkpoint=8..)" yaw="@0" pitch="@0" observers="never" destination="cp-9" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-8, tp_checkpoint=..7)" yaw="@0" pitch="@0" observers="never" destination="cp-8" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-7, tp_checkpoint=7..)" yaw="@0" pitch="@0" observers="never" destination="cp-8" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-7, tp_checkpoint=..6)" yaw="@0" pitch="@0" observers="never" destination="cp-7" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-6, tp_checkpoint=6..)" yaw="@0" pitch="@0" observers="never" destination="cp-7" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-6, tp_checkpoint=..5)" yaw="@0" pitch="@0" observers="never" destination="cp-6" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-5, tp_checkpoint=5..)" yaw="@0" pitch="@0" observers="never" destination="cp-6" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-5, tp_checkpoint=..4)" yaw="@0" pitch="@0" observers="never" destination="cp-5" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-4, tp_checkpoint=4..)" yaw="@0" pitch="@0" observers="never" destination="cp-5" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-4, tp_checkpoint=..3)" yaw="@0" pitch="@0" observers="never" destination="cp-4" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-3, tp_checkpoint=3..)" yaw="@0" pitch="@0" observers="never" destination="cp-4" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-3, tp_checkpoint=..2)" yaw="@0" pitch="@0" observers="never" destination="cp-3" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-2, tp_checkpoint=2..)" yaw="@0" pitch="@0" observers="never" destination="cp-3" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-2, tp_checkpoint=..1)" yaw="@0" pitch="@0" observers="never" destination="cp-2" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-1, tp_checkpoint=1..)" yaw="@0" pitch="@0" observers="never" destination="cp-2" />
-    <portal forward="all(portal-debounce, do_tp=2, lvl-1, tp_checkpoint=0)" yaw="@0" pitch="@0" observers="never" destination="cp-1" />
-    <portal filter="not(participating)" yaw="@0" pitch="@0" region="void" destination="spawn" />
+    <portal forward="all(not(portal-debounce), do_tp=1, tp_lane=1)" x="58" observers="never" sound="false" />
```

Comment:
I'm not sure if this will work, but you should be able to move `sound=false` to the `<portals>` element instead of repeating it for each portal over again.

Reply:
Nice catch! I tried this out with other parameters, and it worked, updated it appropriately.